lib/subjects/designator_selector.rb

```diff
@@ -36,11 +36,7 @@ def get_subjects(user, group_id, limit)
       subject_ids = self.class.client.get_subjects(workflow.id, user.try(&:id), group_id, limit)
       return [] unless subject_ids.present?
       raise "Hacking attempt" unless subject_ids.all? { |i| i.is_a? Integer }
```

Comment:
This `raise` line was in there because of the now removed raw SQL statement underneath it, to guard against a MITM-attack being able to then perform SQL injection. It can go away now.